toucan_connectors/common.py

```diff
@@ -32,6 +32,24 @@ class NonValidVariable(Exception):
     """ Error thrown for a non valid variable in endpoint """
 
 
+def is_jinja_alone(s: str) -> bool:
+    """
+    Return True if the given string is a jinja template alone.
+
+    For example, these strings are jinja templates alone:
+        '{{ foo }}'
+        '{{ foo + bar }}'
+        '{%if my_indic%}1{%else%}2{%endif%}'
```

Comment:
Curiosity: do we have cases where we use such syntax?

Also it's difficult to say it's just jinja alone if it starts and end by {% %}. An example would be:
`'{%if count %}{{ count }}{%else%}No{%endif%} chair{% if count > 1 %}s{% endif %}`'. In that case, keeping a string would be necessary.

Reply:
On a surtout des syntaxes de type `"{{ ... }}"`. La 2e syntaxe je pense pas qu'on en ai des masses. Par contre après vérification le NativeEnvironment jinja gère bien ce cas aussi en fait, donc je vais juste ajouter des tests pour ça.

Reply:
Ajouté dans 93a64e8

Reply:
Excellent that NativeEnv handles it, thanks for the new tests!